Dockerfile.dev

```diff
@@ -44,7 +44,9 @@ RUN mv /app/database-docker/db/ /app/ && rm -rf /app/database-docker/
 RUN echo "host all  all    0.0.0.0/0  md5" >> /etc/postgresql/15/main/pg_hba.conf
 RUN echo "listen_addresses='*'" >> /etc/postgresql/15/main/postgresql.conf
 RUN sed -i "s/^port = .*/port = ${DB_PORT}/" /etc/postgresql/15/main/postgresql.conf
-RUN npm install --global yarn --force
+RUN curl -fsSL https://deb.nodesource.com/setup_22.x | bash - \
+    && apt-get install -y nodejs \
+    && npm install --global yarn
```

Comment:
Why is this required in the dev setup, if we are already using the pre-built base image. This would slow down the dev process.

Reply:
Can you push the pre-built base image with the latest changes so I can remove this one? As the old base image has node 16

Reply:
Push into dockerhub I mean